services/brig/src/Brig/API.hs

```diff
@@ -211,6 +214,15 @@ sitemap o = do
       .&. contentType "application" "json"
       .&. request
 
+    -- provided a list of services, tells you for each whether it's
+    -- whitelisted or not; gets an array of ServiceRefs and returns an array
+    -- of bools
+    post "/i/teams/:tid/services/whitelisted" (continue internalQueryServiceWhitelist) $
```

Comment:
It'd be good to avoid passing around plain lists in JSON as they are a bit inflexible; given that it's only an internal endpoint, accepting a list is probably not too problematic.

Wrt the result type, I'd avoid a plain list of bools since that relies on an order which is a bit fragile and easy to get wrong on the caller side; the overhead of sending back an object with sth like `pid,sid,whitelisted/notwhitelisted` seems worthwhile.